rest/servers/spring/src/main/java/com/connexta/ingest/rest/spring/IngestApi.java

```diff
@@ -75,6 +76,12 @@
               required = true)
           @RequestHeader(value = "Accept-Version", required = true)
           String acceptVersion,
+      @ApiParam(
+              value =
+                  "The RFC 3339, section 5.6 date-time format string of the last modified time of the file(s) to be ingested. ",
```

Comment:
✏️ This description doesn't match the yaml file.

Reply:
Updated